client/src/app/shared/shared-main/misc/channels-setup-message.component.ts

```diff
@@ -16,15 +16,13 @@ export class ChannelsSetupMessageComponent implements OnInit {
     private authService: AuthService
   ) {}
 
-  get userInformationLoaded () {
-    return this.authService.userInformationLoaded
-  }
-
   get hasChannelNotConfigured () {
+    if (!this.user.videoChannels) return
```

Comment:
Sorry, I think it should be `return false`

Reply:
I've re-tried, it seems to work, it's just very long process because I need to re-build for each changes with that bug.